acorn/acorn-tests.ts

```diff
@@ -67,3 +67,6 @@ acorn.getLineInfo('string', 56);
 
 acorn.plugins['test'] = function (p: acorn.Parser, config: any) {
 }
+
+acorn.tokenizer('console.log("hello world)', {locations: true}).getToken();
+acorn.tokenizer('console.log("hello world)', {locations: true})[Symbol.iterator]().next();
```

Comment:
So apparently that only works with `--target=ES6` or higher. Otherwise you get an error:

```
acorn-tests.ts(74,25): error TS2495: Type 'ITokenizer' is not an array type or a string type.
```